modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -16,7 +16,7 @@ class ClaimsAndAppealsController < ApplicationController
 
       before_action { authorize :evss, :access? }
```

Comment:
Do you need to be checking for evss access if the flipper is flipped and it's using lighthouse?

Reply:
good catch. fixed.